features/docs/gegeven-stap-definities-gezag-expressief.feature

```diff
@@ -39,3 +39,23 @@ Functionaliteit: Stap definities ten behoeve van specificeren gezagsrelaties
       En heeft de persoon 'Arjan' de volgende rijen in tabel 'lo3_pl_persoon'
         | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr | geslachts_naam |akte_nr | geboorte_land_code |
         |     1 | P            |         0 |       0 |         000000024 | Arjan          |        |                    |
+
```

Comment:
deze scenario verplaatsen naar gegeven-stap-definities-partner.feature. Dan is het makkelijker terug te vinden